polkadot/runtime/westend/src/lib.rs

```diff
@@ -1559,6 +1569,10 @@ pub mod migrations {
 		pallet_referenda::migration::v1::MigrateV0ToV1<Runtime, ()>,
 		pallet_nomination_pools::migration::versioned_migrations::V6ToV7<Runtime>,
 		pallet_grandpa::migrations::MigrateV4ToV5<Runtime>,
+		frame_support::migrations::RemovePallet<
```

Comment:
`RemovePallet` documentation [warns](https://github.com/paritytech/polkadot-sdk/blob/54ca4f131b82f45b0c2d1f316d65d7c97ad9a99b/substrate/frame/support/src/migrations.rs#L287) about exceeding the block weight limit. Considering the size of `im-online` storage (not too big IIUC) and 50% constant used in `SessionKeys` migration, couldn't it cause problems on production networks?

Reply:
You are missing here the key migration.

Reply:
> You are missing here the key migration.

It's several lines higher. It was already there because of Beefy migration, and I decided not to move it.